### ISSUE TITLE
47%  VS 1.3%  college attendance rate in Beijing VS rural China

### ISSUE THREAD

**Issue by shuyao1201** [jsoma/playfair-projects#95]
# Summary

In the Beijing-Tianjin-Hebei region，470 out of 1000 elementry students can make it to college , while the numer is 13 in rural China, according to the Rural Education Action Program, Stanford University.

I'm trying to make detailed graphs about college attendance among more regions in China, comparing the urban area and rural area. And partial reasons contributed to the situation. 

# Data Source:
Rural Education Action Program:
http://reap.fsi.stanford.edu/research/organization/6084/67097

Inequalities in the Pathway to College in China: When Do Students from Poor Areas Fall Behind?
http://reap.fsi.stanford.edu/sites/default/files/277-inequality_pathway_to_college.pdf

Information, college decisions and financial aid:Evidence from a cluster-randomized controlled trial in China
http://reap.fsi.stanford.edu/sites/default/files/Effects_financial_aid_information_China.pdf

College is a Rich, Han, Urban, Male Club:Research Notes from a Census Survey of Four Tier
One Colleges in China
http://reap.fsi.stanford.edu/sites/default/files/s0305741013000647a.pdf

# Problems I have now: 
I only found the pdf documents, also some data frames inside the pdf files. So I'm try to build th datasets manually, and email the author to ask for raw data at the same time. Is there better ways?

# Pitch issue checklist

- [x] My pitch issue has `[Pitch]` in the title
- [ ] My pitch issue has `[Data request]` in the title if I need additional data
- [x] My pitch issue links to my data set
- [x] My pitch issue explains what I'd like to explore in the data set
- [x] My pitch issue includes multiple possible headlines
- [ ] My pitch issue includes any links/images I'm using as inspiration
- [x] I have received two comments of peer feedback on my pitch issue

**Comment by barjacks** [comment]
I'd use Tabula to extract the data from PDFs. http://tabula.technology/

**Comment by thisss** [comment]
Great idea. I guess, you should not only make graphs, but also -- a map. ;-)

**Comment by jowang0319** [comment]
Maybe you can can try to find some dataset directly from China, there's some times some difference.. and there's a chance that you can get more detailed information

**Comment by jsoma** [comment]
Tabula is great for extracting the numbers from PDFs, and from looking at your PDFs it seems like they're mostly Tabula-able.

On the other hand, the tables aren't really that large! If you find one you think is worth visualizing, spending 15 minutes copying down the numbers into Excel wouldn't be fun but it's probably faster than getting Tabula to do everything right 👍 

I think this is going to be a great project for putting descriptive, "here is what my story is" titles on visualizations instead of vague ones like "% enrollment"

**Comment by shuyao1201** [comment]
![image](https://cloud.githubusercontent.com/assets/19495692/17419146/1d0352cc-5a6a-11e6-92e9-ae6353f108b4.png)

Thanks for your advice of mapping. I do want to, but I don't know how to do it. I'm sure I will learn it later. And it asks for data from many provinces, which I failed to find more about it. @thisss 

Thanks a lot for you suggests@jsoma . I've made datasets by hands and graphed three charts of this pitch, although it is an inadequate description.

@jowang0319 Thank you Jo! I tried to find more data from China's provincial educational department. Unfortunately, it failed. So I just use the inadequate data both from PDFs and other news website.
